### ISSUE TITLE
Runaway exception St13 runtime

### ISSUE THREAD

**Issue by ltm2012** [obsidianproject/Obsidian-Qt#55]

**Comment by ltm2012** [comment]
**Windows 7/system info**
![img_0042](https://user-images.githubusercontent.com/35662633/35196371-65d8f730-fe8e-11e7-972d-87229f8ef767.JPG)
**The Issue (discovered 1-20-18):**
![img_0040](https://user-images.githubusercontent.com/35662633/35196349-13e9f000-fe8e-11e7-81b5-4a778061eeaf.JPG)

**Obsidian Wallet info(worked perfectly until I tried running it yesterday):**
![img_0041](https://user-images.githubusercontent.com/35662633/35196361-3eda4530-fe8e-11e7-9627-e7853a2f310d.JPG)

For some reason, the main folder of the wallet when I first downloaded it on December 2017, to extract all files along with wallet dat are missing.  I don't remember deleting it.  I have run EaseUS Data Recovery Wizard to be sure.... but no file found.  I also see that a new version has come out for windows 10, but not sure if that has anything to do with the issue.  Any help, guidance, suggestions, comments, are appreciated- ltm2012( new github user)
